app/assets/javascripts/school-search/data.js

```diff
@@ -0,0 +1,97 @@
+export const allSchools = [
+  {
+    urn: 123456,
+    name: 'Corfe Hills School',
+    address: 'Higher Blandford Road, Broadstone, Dorset, BH18 9BG',
```

Comment:
Could be useful to separate city and postcode for further filtering

Reply:
Location search was out of spec for the prototype, this is just for display. I can though, if you think it's worth it?